[2022-10-03 08:15:11,423] INFO [GroupCoordinator 0]: Preparing to rebalance group myGroup in state PreparingRebalance with old generation 3 (__consumer_offsets-17) (reason: Removing member consumer-myGroup-1-cdebc2b7-6888-49d2-8053-a030eab4ed14 on LeaveGroup; client reason: not provided) (kafka.coordinator.group.GroupCoordinator)
[2022-10-03 08:15:11,424] INFO [GroupCoordinator 0]: Group myGroup with generation 4 is now empty (__consumer_offsets-17) (kafka.coordinator.group.GroupCoordinator)
[2022-10-03 08:15:11,425] INFO [GroupCoordinator 0]: Member MemberMetadata(memberId=consumer-myGroup-1-cdebc2b7-6888-49d2-8053-a030eab4ed14, groupInstanceId=None, clientId=consumer-myGroup-1, clientHost=/127.0.0.1, sessionTimeoutMs=45000, rebalanceTimeoutMs=300000, supportedProtocols=List(range, cooperative-sticky)) has left group myGroup through explicit `LeaveGroup`; client reason: not provided (kafka.coordinator.group.GroupCoordinator)
[2022-10-03 08:19:13,939] INFO [GroupMetadataManager brokerId=0] Group myGroup transitioned to Dead in generation 4 (kafka.coordinator.group.GroupMetadataManager)
[2022-10-03 08:26:50,678] INFO [GroupCoordinator 0]: Dynamic member with unknown member id joins group myGroup in Empty state. Created a new member id consumer-myGroup-1-143cbcc3-b568-4ea3-91be-b15a12b8e3b1 and request the member to rejoin with this id. (kafka.coordinator.group.GroupCoordinator)
[2022-10-03 08:26:50,681] INFO [GroupCoordinator 0]: Preparing to rebalance group myGroup in state PreparingRebalance with old generation 0 (__consumer_offsets-17) (reason: Adding new member consumer-myGroup-1-143cbcc3-b568-4ea3-91be-b15a12b8e3b1 with group instance id None; client reason: not provided) (kafka.coordinator.group.GroupCoordinator)
[2022-10-03 08:26:50,681] INFO [GroupCoordinator 0]: Stabilized group myGroup generation 1 (__consumer_offsets-17) with 1 members (kafka.coordinator.group.GroupCoordinator)
[2022-10-03 08:26:50,687] INFO [GroupCoordinator 0]: Assignment received from leader consumer-myGroup-1-143cbcc3-b568-4ea3-91be-b15a12b8e3b1 for group myGroup for generation 1. The group has 1 members, 0 of which are static. (kafka.coordinator.group.GroupCoordinator)
[2022-10-03 08:27:59,404] WARN [SocketServer listenerType=ZK_BROKER, nodeId=0] Unexpected error from /127.0.0.1 (channelId=127.0.0.1:9092-127.0.0.1:52224-3); closing connection (org.apache.kafka.common.network.Selector)
org.apache.kafka.common.network.InvalidReceiveException: Invalid receive (size = 1195725856 larger than 104857600)
	at org.apache.kafka.common.network.NetworkReceive.readFrom(NetworkReceive.java:105)
	at org.apache.kafka.common.network.KafkaChannel.receive(KafkaChannel.java:452)
	at org.apache.kafka.common.network.KafkaChannel.read(KafkaChannel.java:402)
	at org.apache.kafka.common.network.Selector.attemptRead(Selector.java:674)
	at org.apache.kafka.common.network.Selector.pollSelectionKeys(Selector.java:576)
	at org.apache.kafka.common.network.Selector.poll(Selector.java:481)
	at kafka.network.Processor.poll(SocketServer.scala:1144)
	at kafka.network.Processor.run(SocketServer.scala:1047)
	at java.base/java.lang.Thread.run(Thread.java:833)
[2022-10-03 08:27:59,409] WARN [SocketServer listenerType=ZK_BROKER, nodeId=0] Unexpected error from /127.0.0.1 (channelId=127.0.0.1:9092-127.0.0.1:52230-4); closing connection (org.apache.kafka.common.network.Selector)
org.apache.kafka.common.network.InvalidReceiveException: Invalid receive (size = 1195725856 larger than 104857600)
	at org.apache.kafka.common.network.NetworkReceive.readFrom(NetworkReceive.java:105)
	at org.apache.kafka.common.network.KafkaChannel.receive(KafkaChannel.java:452)
	at org.apache.kafka.common.network.KafkaChannel.read(KafkaChannel.java:402)
	at org.apache.kafka.common.network.Selector.attemptRead(Selector.java:674)
	at org.apache.kafka.common.network.Selector.pollSelectionKeys(Selector.java:576)
	at org.apache.kafka.common.network.Selector.poll(Selector.java:481)
	at kafka.network.Processor.poll(SocketServer.scala:1144)
	at kafka.network.Processor.run(SocketServer.scala:1047)
	at java.base/java.lang.Thread.run(Thread.java:833)
[2022-10-03 08:27:59,410] WARN [SocketServer listenerType=ZK_BROKER, nodeId=0] Unexpected error from /127.0.0.1 (channelId=127.0.0.1:9092-127.0.0.1:52234-4); closing connection (org.apache.kafka.common.network.Selector)
org.apache.kafka.common.network.InvalidReceiveException: Invalid receive (size = 1195725856 larger than 104857600)
	at org.apache.kafka.common.network.NetworkReceive.readFrom(NetworkReceive.java:105)
	at org.apache.kafka.common.network.KafkaChannel.receive(KafkaChannel.java:452)
	at org.apache.kafka.common.network.KafkaChannel.read(KafkaChannel.java:402)
	at org.apache.kafka.common.network.Selector.attemptRead(Selector.java:674)
	at org.apache.kafka.common.network.Selector.pollSelectionKeys(Selector.java:576)
	at org.apache.kafka.common.network.Selector.poll(Selector.java:481)
	at kafka.network.Processor.poll(SocketServer.scala:1144)
	at kafka.network.Processor.run(SocketServer.scala:1047)
	at java.base/java.lang.Thread.run(Thread.java:833)
[2022-10-03 08:27:59,411] WARN [SocketServer listenerType=ZK_BROKER, nodeId=0] Unexpected error from /127.0.0.1 (channelId=127.0.0.1:9092-127.0.0.1:52244-4); closing connection (org.apache.kafka.common.network.Selector)
org.apache.kafka.common.network.InvalidReceiveException: Invalid receive (size = 1195725856 larger than 104857600)
	at org.apache.kafka.common.network.NetworkReceive.readFrom(NetworkReceive.java:105)
	at org.apache.kafka.common.network.KafkaChannel.receive(KafkaChannel.java:452)
	at org.apache.kafka.common.network.KafkaChannel.read(KafkaChannel.java:402)
	at org.apache.kafka.common.network.Selector.attemptRead(Selector.java:674)
	at org.apache.kafka.common.network.Selector.pollSelectionKeys(Selector.java:576)
	at org.apache.kafka.common.network.Selector.poll(Selector.java:481)
	at kafka.network.Processor.poll(SocketServer.scala:1144)
	at kafka.network.Processor.run(SocketServer.scala:1047)
	at java.base/java.lang.Thread.run(Thread.java:833)
[2022-10-03 08:27:59,412] WARN [SocketServer listenerType=ZK_BROKER, nodeId=0] Unexpected error from /127.0.0.1 (channelId=127.0.0.1:9092-127.0.0.1:52258-5); closing connection (org.apache.kafka.common.network.Selector)
org.apache.kafka.common.network.InvalidReceiveException: Invalid receive (size = 1195725856 larger than 104857600)
	at org.apache.kafka.common.network.NetworkReceive.readFrom(NetworkReceive.java:105)
	at org.apache.kafka.common.network.KafkaChannel.receive(KafkaChannel.java:452)
	at org.apache.kafka.common.network.KafkaChannel.read(KafkaChannel.java:402)
	at org.apache.kafka.common.network.Selector.attemptRead(Selector.java:674)
	at org.apache.kafka.common.network.Selector.pollSelectionKeys(Selector.java:576)
	at org.apache.kafka.common.network.Selector.poll(Selector.java:481)
	at kafka.network.Processor.poll(SocketServer.scala:1144)
	at kafka.network.Processor.run(SocketServer.scala:1047)
	at java.base/java.lang.Thread.run(Thread.java:833)
[2022-10-03 08:27:59,414] WARN [SocketServer listenerType=ZK_BROKER, nodeId=0] Unexpected error from /127.0.0.1 (channelId=127.0.0.1:9092-127.0.0.1:52272-5); closing connection (org.apache.kafka.common.network.Selector)
org.apache.kafka.common.network.InvalidReceiveException: Invalid receive (size = 1195725856 larger than 104857600)
	at org.apache.kafka.common.network.NetworkReceive.readFrom(NetworkReceive.java:105)
	at org.apache.kafka.common.network.KafkaChannel.receive(KafkaChannel.java:452)
	at org.apache.kafka.common.network.KafkaChannel.read(KafkaChannel.java:402)
	at org.apache.kafka.common.network.Selector.attemptRead(Selector.java:674)
	at org.apache.kafka.common.network.Selector.pollSelectionKeys(Selector.java:576)
	at org.apache.kafka.common.network.Selector.poll(Selector.java:481)
	at kafka.network.Processor.poll(SocketServer.scala:1144)
	at kafka.network.Processor.run(SocketServer.scala:1047)
	at java.base/java.lang.Thread.run(Thread.java:833)
[2022-10-03 08:27:59,416] WARN [SocketServer listenerType=ZK_BROKER, nodeId=0] Unexpected error from /127.0.0.1 (channelId=127.0.0.1:9092-127.0.0.1:52276-5); closing connection (org.apache.kafka.common.network.Selector)
org.apache.kafka.common.network.InvalidReceiveException: Invalid receive (size = 1195725856 larger than 104857600)
	at org.apache.kafka.common.network.NetworkReceive.readFrom(NetworkReceive.java:105)
	at org.apache.kafka.common.network.KafkaChannel.receive(KafkaChannel.java:452)
	at org.apache.kafka.common.network.KafkaChannel.read(KafkaChannel.java:402)
	at org.apache.kafka.common.network.Selector.attemptRead(Selector.java:674)
	at org.apache.kafka.common.network.Selector.pollSelectionKeys(Selector.java:576)
	at org.apache.kafka.common.network.Selector.poll(Selector.java:481)
	at kafka.network.Processor.poll(SocketServer.scala:1144)
	at kafka.network.Processor.run(SocketServer.scala:1047)
	at java.base/java.lang.Thread.run(Thread.java:833)
[2022-10-03 08:27:59,417] WARN [SocketServer listenerType=ZK_BROKER, nodeId=0] Unexpected error from /127.0.0.1 (channelId=127.0.0.1:9092-127.0.0.1:52286-6); closing connection (org.apache.kafka.common.network.Selector)
org.apache.kafka.common.network.InvalidReceiveException: Invalid receive (size = 1195725856 larger than 104857600)
	at org.apache.kafka.common.network.NetworkReceive.readFrom(NetworkReceive.java:105)
	at org.apache.kafka.common.network.KafkaChannel.receive(KafkaChannel.java:452)
	at org.apache.kafka.common.network.KafkaChannel.read(KafkaChannel.java:402)
	at org.apache.kafka.common.network.Selector.attemptRead(Selector.java:674)
	at org.apache.kafka.common.network.Selector.pollSelectionKeys(Selector.java:576)
	at org.apache.kafka.common.network.Selector.poll(Selector.java:481)
	at kafka.network.Processor.poll(SocketServer.scala:1144)
	at kafka.network.Processor.run(SocketServer.scala:1047)
	at java.base/java.lang.Thread.run(Thread.java:833)
[2022-10-03 08:27:59,418] WARN [SocketServer listenerType=ZK_BROKER, nodeId=0] Unexpected error from /127.0.0.1 (channelId=127.0.0.1:9092-127.0.0.1:52296-6); closing connection (org.apache.kafka.common.network.Selector)
org.apache.kafka.common.network.InvalidReceiveException: Invalid receive (size = 1195725856 larger than 104857600)
	at org.apache.kafka.common.network.NetworkReceive.readFrom(NetworkReceive.java:105)
	at org.apache.kafka.common.network.KafkaChannel.receive(KafkaChannel.java:452)
	at org.apache.kafka.common.network.KafkaChannel.read(KafkaChannel.java:402)
	at org.apache.kafka.common.network.Selector.attemptRead(Selector.java:674)
	at org.apache.kafka.common.network.Selector.pollSelectionKeys(Selector.java:576)
	at org.apache.kafka.common.network.Selector.poll(Selector.java:481)
	at kafka.network.Processor.poll(SocketServer.scala:1144)
	at kafka.network.Processor.run(SocketServer.scala:1047)
	at java.base/java.lang.Thread.run(Thread.java:833)
[2022-10-03 08:27:59,418] WARN [SocketServer listenerType=ZK_BROKER, nodeId=0] Unexpected error from /127.0.0.1 (channelId=127.0.0.1:9092-127.0.0.1:52306-6); closing connection (org.apache.kafka.common.network.Selector)
org.apache.kafka.common.network.InvalidReceiveException: Invalid receive (size = 1195725856 larger than 104857600)
	at org.apache.kafka.common.network.NetworkReceive.readFrom(NetworkReceive.java:105)
	at org.apache.kafka.common.network.KafkaChannel.receive(KafkaChannel.java:452)
	at org.apache.kafka.common.network.KafkaChannel.read(KafkaChannel.java:402)
	at org.apache.kafka.common.network.Selector.attemptRead(Selector.java:674)
	at org.apache.kafka.common.network.Selector.pollSelectionKeys(Selector.java:576)
	at org.apache.kafka.common.network.Selector.poll(Selector.java:481)
	at kafka.network.Processor.poll(SocketServer.scala:1144)
	at kafka.network.Processor.run(SocketServer.scala:1047)
	at java.base/java.lang.Thread.run(Thread.java:833)
[2022-10-03 08:34:09,849] WARN [SocketServer listenerType=ZK_BROKER, nodeId=0] Unexpected error from /0:0:0:0:0:0:0:1 (channelId=0:0:0:0:0:0:0:1:9092-0:0:0:0:0:0:0:1:40048-7); closing connection (org.apache.kafka.common.network.Selector)
org.apache.kafka.common.network.InvalidReceiveException: Invalid receive (size = 1347375956 larger than 104857600)
	at org.apache.kafka.common.network.NetworkReceive.readFrom(NetworkReceive.java:105)
	at org.apache.kafka.common.network.KafkaChannel.receive(KafkaChannel.java:452)
	at org.apache.kafka.common.network.KafkaChannel.read(KafkaChannel.java:402)
	at org.apache.kafka.common.network.Selector.attemptRead(Selector.java:674)
	at org.apache.kafka.common.network.Selector.pollSelectionKeys(Selector.java:576)
	at org.apache.kafka.common.network.Selector.poll(Selector.java:481)
	at kafka.network.Processor.poll(SocketServer.scala:1144)
	at kafka.network.Processor.run(SocketServer.scala:1047)
	at java.base/java.lang.Thread.run(Thread.java:833)
[2022-10-03 08:34:22,746] WARN [SocketServer listenerType=ZK_BROKER, nodeId=0] Unexpected error from /0:0:0:0:0:0:0:1 (channelId=0:0:0:0:0:0:0:1:9092-0:0:0:0:0:0:0:1:39016-8); closing connection (org.apache.kafka.common.network.Selector)
org.apache.kafka.common.network.InvalidReceiveException: Invalid receive (size = 369295616 larger than 104857600)
	at org.apache.kafka.common.network.NetworkReceive.readFrom(NetworkReceive.java:105)
	at org.apache.kafka.common.network.KafkaChannel.receive(KafkaChannel.java:452)
	at org.apache.kafka.common.network.KafkaChannel.read(KafkaChannel.java:402)
	at org.apache.kafka.common.network.Selector.attemptRead(Selector.java:674)
	at org.apache.kafka.common.network.Selector.pollSelectionKeys(Selector.java:576)
	at org.apache.kafka.common.network.Selector.poll(Selector.java:481)
	at kafka.network.Processor.poll(SocketServer.scala:1144)
	at kafka.network.Processor.run(SocketServer.scala:1047)
	at java.base/java.lang.Thread.run(Thread.java:833)
[2022-10-03 08:36:37,421] INFO [GroupCoordinator 0]: Preparing to rebalance group myGroup in state PreparingRebalance with old generation 1 (__consumer_offsets-17) (reason: Removing member consumer-myGroup-1-143cbcc3-b568-4ea3-91be-b15a12b8e3b1 on LeaveGroup; client reason: not provided) (kafka.coordinator.group.GroupCoordinator)
[2022-10-03 08:36:37,421] INFO [GroupCoordinator 0]: Group myGroup with generation 2 is now empty (__consumer_offsets-17) (kafka.coordinator.group.GroupCoordinator)
[2022-10-03 08:36:37,422] INFO [GroupCoordinator 0]: Member MemberMetadata(memberId=consumer-myGroup-1-143cbcc3-b568-4ea3-91be-b15a12b8e3b1, groupInstanceId=None, clientId=consumer-myGroup-1, clientHost=/127.0.0.1, sessionTimeoutMs=45000, rebalanceTimeoutMs=300000, supportedProtocols=List(range, cooperative-sticky)) has left group myGroup through explicit `LeaveGroup`; client reason: not provided (kafka.coordinator.group.GroupCoordinator)
[2022-10-03 08:39:04,180] INFO [GroupCoordinator 0]: Dynamic member with unknown member id joins group myGroup in Empty state. Created a new member id consumer-myGroup-1-96f7f502-65c2-43ac-83a2-2a9bdd9bb771 and request the member to rejoin with this id. (kafka.coordinator.group.GroupCoordinator)
[2022-10-03 08:39:04,182] INFO [GroupCoordinator 0]: Preparing to rebalance group myGroup in state PreparingRebalance with old generation 2 (__consumer_offsets-17) (reason: Adding new member consumer-myGroup-1-96f7f502-65c2-43ac-83a2-2a9bdd9bb771 with group instance id None; client reason: not provided) (kafka.coordinator.group.GroupCoordinator)
[2022-10-03 08:39:04,182] INFO [GroupCoordinator 0]: Stabilized group myGroup generation 3 (__consumer_offsets-17) with 1 members (kafka.coordinator.group.GroupCoordinator)
[2022-10-03 08:39:04,187] INFO [GroupCoordinator 0]: Assignment received from leader consumer-myGroup-1-96f7f502-65c2-43ac-83a2-2a9bdd9bb771 for group myGroup for generation 3. The group has 1 members, 0 of which are static. (kafka.coordinator.group.GroupCoordinator)
[2022-10-03 08:40:51,175] INFO [GroupCoordinator 0]: Preparing to rebalance group myGroup in state PreparingRebalance with old generation 3 (__consumer_offsets-17) (reason: Removing member consumer-myGroup-1-96f7f502-65c2-43ac-83a2-2a9bdd9bb771 on LeaveGroup; client reason: not provided) (kafka.coordinator.group.GroupCoordinator)
[2022-10-03 08:40:51,175] INFO [GroupCoordinator 0]: Group myGroup with generation 4 is now empty (__consumer_offsets-17) (kafka.coordinator.group.GroupCoordinator)
[2022-10-03 08:40:51,176] INFO [GroupCoordinator 0]: Member MemberMetadata(memberId=consumer-myGroup-1-96f7f502-65c2-43ac-83a2-2a9bdd9bb771, groupInstanceId=None, clientId=consumer-myGroup-1, clientHost=/127.0.0.1, sessionTimeoutMs=45000, rebalanceTimeoutMs=300000, supportedProtocols=List(range, cooperative-sticky)) has left group myGroup through explicit `LeaveGroup`; client reason: not provided (kafka.coordinator.group.GroupCoordinator)
[2022-10-03 08:41:02,895] INFO [GroupCoordinator 0]: Dynamic member with unknown member id joins group myGroup in Empty state. Created a new member id consumer-myGroup-1-b5415283-df24-4e85-8980-7b644c80945c and request the member to rejoin with this id. (kafka.coordinator.group.GroupCoordinator)
[2022-10-03 08:41:02,897] INFO [GroupCoordinator 0]: Preparing to rebalance group myGroup in state PreparingRebalance with old generation 4 (__consumer_offsets-17) (reason: Adding new member consumer-myGroup-1-b5415283-df24-4e85-8980-7b644c80945c with group instance id None; client reason: not provided) (kafka.coordinator.group.GroupCoordinator)
[2022-10-03 08:41:02,897] INFO [GroupCoordinator 0]: Stabilized group myGroup generation 5 (__consumer_offsets-17) with 1 members (kafka.coordinator.group.GroupCoordinator)
[2022-10-03 08:41:02,902] INFO [GroupCoordinator 0]: Assignment received from leader consumer-myGroup-1-b5415283-df24-4e85-8980-7b644c80945c for group myGroup for generation 5. The group has 1 members, 0 of which are static. (kafka.coordinator.group.GroupCoordinator)
[2022-10-03 08:43:25,532] INFO [GroupCoordinator 0]: Preparing to rebalance group myGroup in state PreparingRebalance with old generation 5 (__consumer_offsets-17) (reason: Removing member consumer-myGroup-1-b5415283-df24-4e85-8980-7b644c80945c on LeaveGroup; client reason: not provided) (kafka.coordinator.group.GroupCoordinator)
[2022-10-03 08:43:25,533] INFO [GroupCoordinator 0]: Group myGroup with generation 6 is now empty (__consumer_offsets-17) (kafka.coordinator.group.GroupCoordinator)
[2022-10-03 08:43:25,534] INFO [GroupCoordinator 0]: Member MemberMetadata(memberId=consumer-myGroup-1-b5415283-df24-4e85-8980-7b644c80945c, groupInstanceId=None, clientId=consumer-myGroup-1, clientHost=/127.0.0.1, sessionTimeoutMs=45000, rebalanceTimeoutMs=300000, supportedProtocols=List(range, cooperative-sticky)) has left group myGroup through explicit `LeaveGroup`; client reason: not provided (kafka.coordinator.group.GroupCoordinator)
[2022-10-03 08:43:59,254] INFO [GroupCoordinator 0]: Dynamic member with unknown member id joins group myGroup in Empty state. Created a new member id consumer-myGroup-1-3992f9dc-1e8a-4569-953b-241c77dd275c and request the member to rejoin with this id. (kafka.coordinator.group.GroupCoordinator)
[2022-10-03 08:43:59,256] INFO [GroupCoordinator 0]: Preparing to rebalance group myGroup in state PreparingRebalance with old generation 6 (__consumer_offsets-17) (reason: Adding new member consumer-myGroup-1-3992f9dc-1e8a-4569-953b-241c77dd275c with group instance id None; client reason: not provided) (kafka.coordinator.group.GroupCoordinator)
[2022-10-03 08:43:59,256] INFO [GroupCoordinator 0]: Stabilized group myGroup generation 7 (__consumer_offsets-17) with 1 members (kafka.coordinator.group.GroupCoordinator)
[2022-10-03 08:43:59,262] INFO [GroupCoordinator 0]: Assignment received from leader consumer-myGroup-1-3992f9dc-1e8a-4569-953b-241c77dd275c for group myGroup for generation 7. The group has 1 members, 0 of which are static. (kafka.coordinator.group.GroupCoordinator)
[2022-10-03 08:51:29,513] INFO [GroupCoordinator 0]: Preparing to rebalance group myGroup in state PreparingRebalance with old generation 7 (__consumer_offsets-17) (reason: Removing member consumer-myGroup-1-3992f9dc-1e8a-4569-953b-241c77dd275c on LeaveGroup; client reason: not provided) (kafka.coordinator.group.GroupCoordinator)
[2022-10-03 08:51:29,513] INFO [GroupCoordinator 0]: Group myGroup with generation 8 is now empty (__consumer_offsets-17) (kafka.coordinator.group.GroupCoordinator)
[2022-10-03 08:51:29,513] INFO [GroupCoordinator 0]: Member MemberMetadata(memberId=consumer-myGroup-1-3992f9dc-1e8a-4569-953b-241c77dd275c, groupInstanceId=None, clientId=consumer-myGroup-1, clientHost=/127.0.0.1, sessionTimeoutMs=45000, rebalanceTimeoutMs=300000, supportedProtocols=List(range, cooperative-sticky)) has left group myGroup through explicit `LeaveGroup`; client reason: not provided (kafka.coordinator.group.GroupCoordinator)
[2022-10-03 08:51:45,097] INFO [GroupCoordinator 0]: Dynamic member with unknown member id joins group myGroup in Empty state. Created a new member id consumer-myGroup-1-489a9d69-13f5-4aeb-a121-8b76d3524ffb and request the member to rejoin with this id. (kafka.coordinator.group.GroupCoordinator)
[2022-10-03 08:51:45,100] INFO [GroupCoordinator 0]: Preparing to rebalance group myGroup in state PreparingRebalance with old generation 8 (__consumer_offsets-17) (reason: Adding new member consumer-myGroup-1-489a9d69-13f5-4aeb-a121-8b76d3524ffb with group instance id None; client reason: not provided) (kafka.coordinator.group.GroupCoordinator)
[2022-10-03 08:51:45,100] INFO [GroupCoordinator 0]: Stabilized group myGroup generation 9 (__consumer_offsets-17) with 1 members (kafka.coordinator.group.GroupCoordinator)
[2022-10-03 08:51:45,105] INFO [GroupCoordinator 0]: Assignment received from leader consumer-myGroup-1-489a9d69-13f5-4aeb-a121-8b76d3524ffb for group myGroup for generation 9. The group has 1 members, 0 of which are static. (kafka.coordinator.group.GroupCoordinator)
